Manifest scientific name warning

emrojo opened the pull request.

NikolausC commented:
Is there something wrong with this PR? There are a load of merge commits before the changes

emrojo commented:
This branch was created from the staging branch, because of that it is carrying all the empty commits that carry the information about the mergings performed in staging from devel. I've rebased and squashed all the commits in one in another branch. It's in pull request https://github.com/sanger/aker-submission/pull/181
